core/lib/basic_types/src/protocol_version.rs

```diff
@@ -87,6 +87,26 @@ impl ProtocolVersionId {
         }
     }
 
+    /// Returns the number of blobs supported by each VM version
+    pub fn into_num_blobs_required(&self) -> usize {
+        if self.is_pre_1_5_0() {
+            2
+        } else {
+            16
+        }
+    }
+
+    /// Returns the number of blobs created within each VM version
+    pub fn into_num_blobs_created(&self) -> usize {
```

Comment:
It's not into. 
And I'd prefer to have 

an additional function somewhere close to blobs creation, where are you based on version return the necessary value. 

Protocol version know nothing about blobs